### PR TITLE
Implementing incomes

### LINKED ISSUE #6: 📦: integrate incomes  with frontend

### PR DESCRIPTION
**Pull request by AndresMorelos**
Will Fix #6

### CODE REVIEW

**Comment by FlavioAandres** [comment]
Nice! I had trhoubles thinking about how to implement this since I don't receive emails when a receive a income. Thanks!! 

Also, may you add this in the chatbot? :3 
I guess after this I could continue the UI for this one

**Comment by AndresMorelos** [comment]
Yeah, the think is manually add the incomes by the UI or Chatbot, in order to generate better graphs.

**Comment by AndresMorelos** [comment]
## Home

image

## Graphs
image

**Comment by AndresMorelos** [comment]
## Adding New Income
image

## Adding Category for Income
image

**Comment by AndresMorelos** [comment]
@FlavioAandres Ready for CR.

**Comment by AndresMorelos** [comment]
@FlavioAandres Using from my side and all is ok.

**Comment by FlavioAandres** [comment]
Nice! If works just merge it, I'll implement it the next week

File: client/src/components/ModalAddIncome/ModalAddIncome.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                <Button onClick={props.close} shape="flat" color="info">
+                    Cancelar
+                </Button>
+                <Button loading={props.loading} color="info" onClick={onCreateIncome}>Crear Ingreso</Button>
+            </Modal.Footer>
+        </Modal>
+    );
+};
+
+export default NewCategoryModal;
```
Comment (FlavioAandres) [inline client/src/components/ModalAddIncome/ModalAddIncome.js:69]: @AndresMorelos I'll fix the name of this in my next PR 😂

**Pull request merged**